Assets/HoloToolkit/Input/Scripts/InputSources/EditorHandsInput.cs

```diff
@@ -326,7 +387,7 @@ private void SendHandVisibilityEvents()
             // Send event for new hands that were added
             foreach (uint newHand in newHands)
             {
-                inputManager.RaiseSourceDetected(this, newHand);
+                InputManager.Instance.RaiseSourceDetected(this, newHand);
```

Comment:
I actually didn't realize I could grab the `inputManager` field from the base class.